js/samples/quickstart-kotlin-azfunction/app/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
no instructions needed, the file is autogenerated by Android studio.
It should not be added to repo because is included in gitignore file, anyway removed.